.github/workflows/codeql.yml

```
@@ -1,13 +1,13 @@
-name: CodeQL
+name: "CodeQL"
 
 on:
   push:
-    branches: [main]
+    branches: [ "main" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)